src/service/operation-id-service.js

```diff
@@ -97,7 +97,9 @@ class OperationIdService {
     async removeOperationIdCache(operationId) {
         this.logger.debug(`Removing operation id: ${operationId} cached data`);
         const operationIdCachePath = this.fileService.getOperationIdDocumentPath(operationId);
-        await this.fileService.removeFile(operationIdCachePath);
+        if (await this.fileService.fileExists(operationIdCachePath)) {
+            await this.fileService.removeFile(operationIdCachePath);
```

Comment:
Can you add check for fileExists in removeFile method that way we don't have to call fileService two times